protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -1319,10 +1320,8 @@ message ExportLedgerEntriesRequest {
     string party_id = 1 [(google.api.field_behavior) = REQUIRED];
     // Asset ID to export ledger entries for.
     string asset_id = 2;
```

Comment:
Only if you want to: if we're making a breaking change here, can we also change:
```
string asset_id = 2 -> optional asset_id = 2
```